modules/core/src/shaderlib/project/project.glsl.js

```diff
@@ -170,6 +170,15 @@ vec4 project_common_position_to_clipspace(vec4 position) {
   return project_uViewProjectionMatrix * position + project_uCenter;
 }
 
+vec4 project_common_position_to_clipspace(mat4 viewProject, vec4 position, vec4 center) {
```

Comment:
- Make `vec4 position` the first argument
- Call this function from `vec4 project_common_position_to_clipspace(vec4 position) ` to avoid duplication of code